R/primary-keys.R

```diff
@@ -7,6 +7,8 @@
 #' If `force == TRUE`, the function will replace an already
 #' set key, without altering foreign keys previously pointing to that primary key.
 #'
+#' @details There can be only one primary key per table in a [`dm`].
```

Comment:
Mention the possibility to add multiple additional unique keys?

Reply:
see 786ee7f737f92be45df3b002e67fc108ab82f62f